stats/stats/src/chart/counters/total_tokens.rs

```diff
@@ -0,0 +1,44 @@
+#![allow(unused_variables)]
+use crate::{chart::insert::insert_double_data, counters_list, UpdateError};
+use async_trait::async_trait;
+use entity::sea_orm_active_enums::{ChartType, ChartValueType};
+use sea_orm::prelude::*;
+
+#[derive(Default, Debug)]
+pub struct TotalTokens {}
+
+#[async_trait]
+impl crate::Chart for TotalTokens {
+    fn name(&self) -> &str {
+        counters_list::TOTAL_TOKENS
+    }
+
+    async fn create(&self, db: &DatabaseConnection) -> Result<(), DbErr> {
+        crate::chart::create_chart(
+            db,
+            self.name().into(),
+            ChartType::Counter,
+            ChartValueType::Double,
```

Comment:
It should be an Int

Reply:
To be honest I didn't understand meaning of total tokens. Is it number of smart contracts of total value of all tokens?

Reply:
Hmm, I've thought that it is the total number of token contracts, but now I'm also not sure. Let ask about that.